### PR TITLE
mm-doc-github-rules.md

### PR DESCRIPTION
**Pull request by mmiele**
This PR addresses the following work items:
- [x] Fix TOC
- [x] Add checklist section

### CODE REVIEW

File: doc-github-rules.md [hunk 1]
```diff
@@ -77,14 +81,32 @@ Once a PR has received sufficient review and concerns are satisfied, a designate
...
+
+## Use checklists for large issues or PRs
+
+When working on a complex issue or PR, it is often preferable to use a checklist within one issue or PR. This avoids the use of multiple issues or PRs that could be difficult to track.  
+
+Especially with an issue related to a feature, all of the pull requests used to implement the feature can refer to the same issue. The progress made can be easily understood by looking at which boxes are checked and which are not. The status is found in one place.
+
+Github even uses this checklist system to print out a small progress summary when looking at the lists of open issues.
+
+You enter the checklist in the comment when you create the issue or the PR. To create a checklist, you use a markdown format similar to the one shown in the example below.
```
Comment (chrispsommers) [inline doc-github-rules.md:96]: What do you do with the checklist as things progress?

File: doc-github-rules.md [hunk 2]
```diff
@@ -1,16 +1,20 @@
 ---
 title: Github basic process
```
Comment (chrispsommers) [inline doc-github-rules.md:2]: I'm not in favor of these header blocks. It takes up space,  doesn't add anything useful - the title should be self-explanatory and the author/date is in the commit info, you're just creating maintenance chores. (Admittedly, if you're not online and reading a printed version, the commit info might not be apparent - but how likely is that?).

**Review by chrispsommers — CHANGES_REQUESTED** [review]

File: doc-github-rules.md (hunk 1 above)
Comment (mmiele) [reply doc-github-rules.md:96]: You check mark the listed work item when completed. I can add this info in the article.

File: doc-github-rules.md (hunk 2 above)
Comment (mmiele) [reply doc-github-rules.md:2]: This is metadata that in online documentation helps in the search. 
Just experimenting with it. 
But I do not know if it helps in our case.

File: doc-github-rules.md [hunk 3]
```diff
@@ -77,14 +81,32 @@ Once a PR has received sufficient review and concerns are satisfied, a designate
 ## Directly Editing Artifacts
 Persons with write access permission or higher can directly edit documents and push changes. This practice should be used very sparingly, and only in the draft stages of documents or to make trivial changes in documents already in use.
 
->**Making direct edits to "released" documents which could impact downstream consumers, without proper review phases, short-circuits the normal community process and can result in breaking changes and significant disruption.**
-> 
+> [!WARNING]
+> Making direct edits to "released" documents which could impact downstream consumers, without proper review phases, short-circuits the normal community process and can result in breaking changes and significant disruption.
```
Comment (KrisNey-MSFT) [inline doc-github-rules.md:85]:
```suggestion
> Direct edits (without proper review phases) to "released" documents short-circuits the normal community process and can result in breaking changes and significant disruption, resulting in potential impact to downstream consumers.
```

File: doc-github-rules.md [hunk 4]
```diff
@@ -77,14 +81,32 @@ Once a PR has received sufficient review and concerns are satisfied, a designate
...
+
+
+## Use checklists for large issues or PRs
+
+When working on a complex issue or PR, it is often preferable to use a checklist within one issue or PR. This avoids the use of multiple issues or PRs that could be difficult to track.
```
Comment (KrisNey-MSFT) [inline doc-github-rules.md:90]:
```suggestion
When working on a complex Issue or PR, it is often preferable to use a checklist within one Issue or PR. This avoids the use of multiple Issues or PRs that could be difficult to track.  
```

File: doc-github-rules.md [hunk 5]
```diff
@@ -77,14 +81,32 @@ Once a PR has received sufficient review and concerns are satisfied, a designate
...
+> Making direct edits to "released" documents which could impact downstream consumers, without proper review phases, short-circuits the normal community process and can result in breaking changes and significant disruption.
+
+
+## Use checklists for large issues or PRs
+
+When working on a complex issue or PR, it is often preferable to use a checklist within one issue or PR. This avoids the use of multiple issues or PRs that could be difficult to track.  
+
+Especially with an issue related to a feature, all of the pull requests used to implement the feature can refer to the same issue. The progress made can be easily understood by looking at which boxes are checked and which are not. The status is found in one place.
+
+Github even uses this checklist system to print out a small progress summary when looking at the lists of open issues.
```
Comment (KrisNey-MSFT) [inline doc-github-rules.md:94]:
```suggestion
The checklist system is also useful to track work, such as a progress summary, when reviewing lists of open Issues.
```

File: doc-github-rules.md (hunk 1 above)
Comment (KrisNey-MSFT) [inline doc-github-rules.md:96]:
```suggestion
A contributor would enter the checklist in the comment when creating the Issue or PR. To create a checklist, please use a markdown format similar to the one shown in the example below.
```

**Review by KrisNey-MSFT — COMMENTED** [review]
Made a few wording changes if you all agree

File: doc-github-rules.md (hunk 1 above)
Comment (mmiele) [reply doc-github-rules.md]: Added NOTE to clarify.

File: doc-github-rules.md (hunk 2 above)
Comment (mmiele) [reply doc-github-rules.md]: Removed metadata.

**Review by chrispsommers — APPROVED** [review]

**Review by KrisNey-MSFT — APPROVED** [review]

**Pull request merged**